src/main/kotlin/com/autonomousapps/model/Coordinates.kt

```diff
@@ -5,59 +5,118 @@ import dev.zacsweers.moshix.sealed.annotations.TypeLabel
 
 @JsonClass(generateAdapter = false, generator = "sealed:type")
 sealed class Coordinates(
-  open val identifier: String
+  open val identifier: String,
+  open val gradleVariantIdentification: GradleVariantIdentification
```

Comment:
Should we call the field just `gvi` (or smth completely different) to make this more compact in the Json files?

Reply:
I don't have a strong opinion either. I also don't have a concrete case where anyone "complained". I just thought that the size of the files were a concern in the past already. I remember you thinking about alternative formats (but maybe I remember that wrongly).

Let's leave it like this for now then.